pkg/model/serviceinstance.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Istio Authors
```

Comment:
Please use camelCase for file names:

serviceinstance.go -> serviceInstance.go